src/components/modals/WcSmartContractModal.tsx

```diff
@@ -58,7 +58,9 @@ export const WcSmartContractModal = (props: Props) => {
 
   const handleSubmit = () => {
     wcRequestResponse(wallet, uri, true, payload)
-      .then(async () => await Airship.show(bridge => <FlashNotification bridge={bridge} message={lstrings.wc_smartcontract_confirmed} />))
+      .then(() => {
+        Airship.show(bridge => <FlashNotification bridge={bridge} message={lstrings.wc_smartcontract_confirmed} />)
```

Comment:
This will be an ESLint error once we enable stricter promise handling. Better add `.catch(()=>{})` on the `Airship.show`, since we don't care about the Airship result.